apps/lecture/crawlers/base.py

```diff
@@ -0,0 +1,62 @@
+import logging
+import time
+from abc import ABC, abstractmethod
+from typing import Any, Dict, Optional
+
+import requests
+from bs4 import BeautifulSoup
+
+# 프로그램이 실행되는 동안 익셉트 발생시 에러기록 (콘솔에기록<터미널/화면>)
+logger = logging.getLogger(__name__)
+
+
+# 크롤링 장비 준비
+class BaseCrawler(ABC):
+
+    def __init__(self, base_url: str, auth_cookie: Optional[str] = None) -> None:
+        self.base_url = base_url
+        self.session = requests.Session()
+        self.headers = {
+            "User-Agent": "Chrome/120.0.6099.71 (Windows NT 10.0; Win64; x64)",
+        }
```

Comment:
여기도 User-Agent 필터링 우회할 수 있도록 조금 더 사람같이
```suggestion
    self.headers = {
        "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) "
                      "AppleWebKit/537.36 (KHTML, like Gecko) "
                      "Chrome/128.0.0.0 Safari/537.36"
    }
```